hypershift-operator/metrics_test.go

```diff
@@ -137,3 +145,61 @@ func TestMetrics(t *testing.T) {
 		})
 	}
 }
+
+func TestGetOperatorImage(t *testing.T) {
+	t.Parallel()
+	testCases := []struct {
+		name            string
+		podName         string
+		hypershiftPod   *corev1.Pod
+		expectedImage   string
+		expectedImageID string
+	}{
+		{
+			name:    "Get hypershift operator image details from pod",
```

Comment:
convetion for naming is "when/if... should..." e.g. "When the pod is running it should gets details from it"
And the other test case we should have is "When the pod is not running it should..."